src/components/ErrorHandlers/Web3ErrorHandler.tsx

```diff
@@ -17,7 +17,8 @@ const Web3ErrorHandler: React.FC<Props> = ({ error }) => {
       case ethers.errors.UNPREDICTABLE_GAS_LIMIT:
         return "Gas limit unpredictable";
       default:
-        return "Unknown error encountered";
+        console.error(error);
```

Comment:
The error is already logged in the console as standard behaviour - i believe this will cause it to be logged twice